src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/RecFlowEvent.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions
+
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.RecFlowEventModel
+
+data class RecFlowEvent(
+  val id: String? = null,
+  val crn: String?,
+  val userId: String?,
+  val timeStamp: String?,
+  val eventType: EventType?
+)
+fun RecFlowEvent.toRecFlowEventModel(): RecFlowEventModel {
+  return RecFlowEventModel(
+    crn = crn,
```

Comment:
I wonder, if we have the recommendation ID, if we should send that as well. Maybe ok without it. What do you think?